### ISSUE TITLE
provide an implementation without jetty

### ISSUE THREAD

**Issue by davidB** [davidB/livereload-jvm#3]
try http://docs.oracle.com/javase/6/docs/jre/api/net/httpserver/spec/com/sun/net/httpserver/package-summary.html

**Comment by PerfectCarl** [comment]
It would fix issue #7 and remove a big dependency...

**Comment by davidB** [comment]
Yep, but I didn't find the motivation to mad it ;-). Someone using livereload-jvm is a good motivation. I should be able to find time to work on it this week / month.

**Comment by PerfectCarl** [comment]
After trying livereload on other stacks (js mostly) I found out that I can't live without anymore (or don't want to anyway).
I'm surprised that this is not more integrated into jvm web stacks.

That being said my issues were about livereloadj-jvm not be compatible with jetty 9.2 (as my stack uses it) and thus introduced conflicts. 

In conclusion with a version that supports jetty 9.2, I am perfectly happy.
A jetty less version would be insanely cool though (I have no idea how hard it would be to get it done)

**Comment by PerfectCarl** [comment]
As a sidenote, why did you use the weird `net_alchim31_livereload` namespace instead of something like `net.alchim31.livereload`
